src/hotspot/share/compiler/compilerThread.hpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 1997, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should a new file only have a single copyright year? I'm not sure what the rule is about splitting existing files but you use a single year in the new cpp files.

Reply:
I think the rule has been: if it's a new file, it gets a new year.

Reply:
Hi David, you're correct that I just cut-and-pasted the code; the only exception was I made the `thread_entry` functions member functions.

I don't have a better idea for the file names, either. I'll keep them as is in the patch. Maybe someone else with a better sense for naming could fix our file names in the future.

I fixed the copyright year.

Reply:
> I think the rule has been: if it's a new file, it gets a new year.

My understanding is that if code has been copied, then the old copyright dates should be kept.